roles-in-tech/machine-learning.md

```diff
@@ -30,10 +30,11 @@ Whether it’s something it acquired or developed in-house, Google has launched
 In this instance, the machine – or algorithm – was developed by DeepMind, which Google bought a few years ago for about $525 million.
 ### 3. Baidu
  - Another company which has been very active on the mergers and acquisitions scene is Chinese search giant Baidu.
-The company is particularly interested in natural language processing, with a view to developing a workable voice-activated search function. One of the many machine learning acquisitions Baidu made last year was [Kitt.ai](http://kitt.ai/), which is said to have a portfolio of chatbots and voice-based applications. The financial terms of that particular deal were not disclosed but Baidu is said to be the 10th biggest-spender on acquisitions in the world, according to CB Insights, which says Google spends the most.
+The company is particularly interested in natural language processing, with a view to developing a workable voice-activated search function. One of the many machine learning acquisitions Baidu made last year was [Kitt.ai](http://kitt.ai/), which is said to have a portfolio of chatbots and voice-based applications. The financial terms of that particular deal were not disclosed but Baidu is said to be the 10th biggest-spender on acquisitions in the world, according to CB Insights, which says Google spends the most.[(em360tech.com)](https://www.em360tech.com/tech-news/top-ten/top-10-companies-using-machine-learning/)
```

Comment:
Do the prerequisites, motivation, or ibm sections have their source?

Reply:
@alodahl IBM  and Google have the same source as that of the others. Shall I mention again individually?
The motivation section was not written by me, it was already there. and prerequisites have the source.

Reply:
Yes, mention ibms individually... linked to its own part of the page if you can.

The prereq has a link, but does the paragraph that it is in come from that source?

Reply:
yes, the link mentioned in prerequisites is for its description. there are 2 links one for the definition of data analysis and other from where the paragraph is cited.

Reply:
and about individual links, its all in the same page and doesn't have specific individual links. So I am mentioning the same in all.